t/op/array.t

```diff
@@ -440,14 +440,13 @@ $::ra = [ bless [], 'A' ];
 pass 'no crash when freeing array that is being cleared';
 
 # [perl #85670] Copying magic to elements
-SKIP: {
-    skip "no Scalar::Util::weaken on miniperl", 1, if is_miniperl;
-    require Scalar::Util;
+{
```

Comment:
This outer bracing is no longer needed I think and should go

Reply:
Possibly; though I didn't want to disturb more lines than necessary, but I might as well for consistency. Done e076c0cd9a